examples/connext_dds/basic_security/README.md

```diff
@@ -0,0 +1,86 @@
+(location of libcrypto.so and libssl.so).
+If you are compiling against the Security Plugins for use with wolfSSL, your
+LD_LIBRARY_PATH must include $NDDSHOME/third_party/wolfssl-<version>/<architecture>/<release or debug>/lib
+(location of libwolfssl.so).
+
+If using Certicom Security Builder Engine for QNX, your LD_LIBRARY_PATH must include
+$NDDSHOME/third_party/openssl-<version>/<architecture>/release/lib/:$CERTICOM_SBENGINEHOME/tools/sb/sb-$CERTICOMOS/lib/:$CERTICOM_SBENGINEHOME/lib/$CERTICOMOS.
+
+To run this example, type the following commands in two different command shells (one command in each shell), either
+on the same machine or on different machines:
+
+  > python3 secure_pub.py
+  > python3 secure_sub.py
+
+Windows systems
+---------------
+If using dynamic libraries, your PATH must include
+%NDDSHOME%\lib\<architecture> and %NDDSHOME%\third_party\openssl-<version>\<architecture>\<release or debug>\bin
+(location of the libcrypto and libssl DLLs).
+
+To run this example, type the following commands in two different command shells (one command in each shell), either
+on the same machine or on different machines:
+
+  > python3 secure_pub.py
+  > python3 secure_sub.py
+
+Accepted parameters
+-------------------
+
+The following parameters are accepted:
+* -d for the domain id
+* -c for the number of samples to send
+* -p for the profile (A, B, RSA_A, RSA_B, ECDSA_P384_A, or ECDSA_P384_B)
+
+
+To run this example
+using the "rsa"(**) algorithm suite, add a "-p" command-line argument: "rsa".
+Pass "p384" (***) for the "-p" command-line argument if you want to use the
+"ecdsa secp384r1" algorithm suite.
+
+* The "ecdsa" algorithm suite consists of ECDSA+P256+SHA256 (digital signature
+  for identity trust chain and authentication) and ECDHE-CEUM+P256 (key establishment).
+** The "rsa" algorithm suite consists of RSASSA-PKCS1-V1_5+2048+SHA256 (digital
+   signature for identity trust chain), RSASSA-PSS-MGF1SHA256+2048+SHA256
+   (digital signature for authentication) and ECDHE-CEUM+P256 (key
+   establishment).
+*** The "ecdsa secp384r1" algorithm suite consists of ECDSA+P384+SHA384 (digital
+   signature for identity trust chain and authentication), and ECDHE-CEUM+P384
+   (key establishment).
+
+Troubleshooting this Example
+============================
+If you see the following error:
+
+Cryptography_encrypt:error:0607B083:lib(6):func(123):reason(131)
+
+you are likely linking against an older version of OpenSSL. You need version
+1.0.1c or later.
+
+For more information, please consult the "RTI Security Plugins Getting Started
+Guide".
+
+
+Differences between the Python Version and other APIs
+=====================================================
+
+In order to accommodate the file location changes the profile in XML
+was modified so that the file locations are correct. Additionally, the
+profile file name has been changed to SecureQos.xml because QoS profiles
```

Comment:
can you explain what you're trying to say here?

is there anything the user needs to know? Otherwise this is going to cause confusion more than anything?

Reply:
I was trying to say that you cannot just drop this file into the other APIs but after thinking a little I think I can remove this section